bin/disco_elasticache.py

```diff
@@ -4,12 +4,14 @@
 
 Usage:
     disco_elasticache.py [--debug] [--env ENV] list
-    disco_elasticache.py [--debug] [--env ENV] update [--cluster CLUSTER]
+    disco_elasticache.py [--debug] [--env ENV] listsnapshots [--cluster CLUSTER]
```

Comment:
I think the convention has been to separate subcommands with dashes like `list-snapshots`